python/lsst/meas/algorithms/__init__.py

```diff
@@ -62,6 +62,7 @@
 from .scaleVariance import *
 from .noise_covariance import *
 from .gp_interpolation import *
+from .computeExPsf import *
```

Comment:
Let's avoid this. Older modules are imported by default to not break things, but newer modules are expected to be specifically imported if required.